src/common/hooks/use-asset-transfer.ts

```diff
@@ -53,21 +56,26 @@ export function useMakeAssetTransfer() {
       return contract.startsWith(contractAddress);
     });
 
+    const realAmount =
+      selectedAsset.type === 'ft'
+        ? ftUnshiftDecimals(amount, selectedAsset?.meta?.decimals || 0)
+        : amount;
+
     const postConditionOptions = tokenBalanceKey
       ? {
           contractAddress,
           contractName,
           assetName,
           stxAddress,
-          amount,
+          amount: realAmount,
         }
       : undefined;
 
     const postConditions = postConditionOptions ? [makePostCondition(postConditionOptions)] : [];
 
     // (transfer (uint principal principal) (response bool uint))
     const functionArgs = [
-      uintCV(amount),
+      uintCV(realAmount),
```

Comment:
What does `CV` refer to?

Reply:
Clarity Value, these are helpers to convert values from js to use in clarity